core/src/io/Dataparser.cpp

```diff
@@ -1059,32 +1070,31 @@ namespace IO
             myfile.Read_String( ovf_data_representation, "# Begin: Data" );
             std::istringstream repr( ovf_data_representation );
             repr >> ovf_data_representation;
-            if( ovf_data_representation == "Binary" ) 
+            if( ovf_data_representation == "binary" ) 
                 repr >> ovf_binary_length;
             
             Log( lvl, sender, fmt::format( "# OVF data representation = {}", ovf_data_representation ) );
             Log( lvl, sender, fmt::format( "# OVF binary length       = {}", ovf_binary_length ) );
             
             // Check that representation and binary length valures are ok
-            if( ovf_data_representation != "Text" && ovf_data_representation != "Binary" )
+            if( ovf_data_representation != "text" && ovf_data_representation != "binary" )
             {
                 spirit_throw(Utility::Exception_Classifier::Bad_File_Content, Utility::Log_Level::Error,
-                    "Data representation must be either \"Text\" or \"Binary\"");
+                    "Data representation must be either \"text\" or \"binary\"");

```

Comment:
This is not right - the keywords *should* be in uppercase, i.e. "Text" or "Binary", according to how MuMax writes OVF files.

Reply:
MuMax is not following the official OVF 2.0 format described in NIST page ( see http://math.nist.gov/oommf/doc/userguide12a5/userguide/OVF_2.0_format.html ). Of course by applying de-capitalization in functions `GetLine()`, `Read_Single()` and `Read_String()` we solve the problem of capitalization and we expect MuMax to be parsed in without any problems.